common/hooks/post-install/99-pkglint-warn-cross-cruft.sh

```diff
@@ -0,0 +1,21 @@
+# This hook warns if :
+# - Any text file /usr/{bin,lib,libexec,share} contains $XBPS_CROSS_BASE
+# - Any text file /usr/{bin,lib,libexec,share} contains $XBPS_WRAPPERDIR
+
+hook() {
+	if [ -z "$CROSS_BUILD" ]; then
+		return 0
+	fi
+	for d in bin lib libexec share; do
+		for f in $PKGDESTDIR/usr/$d/* $PKGDESTDIR/usr/$d/**/*; do
+			case "$(file -bi "$f")" in
+			text/*) if grep -q -e "$XBPS_CROSS_BASE" \
+					   -e "$XBPS_WRAPPERDIR" "$f"; then
```

Comment:
Maybe also `$XBPS_CROSS_TRIPLET`?

Reply:
I don't know.

```sh
git grep 'sed.*XBPS_CROSS_TRIP' srcpkgs/
```

returns no interesting use-cases.
Only used in language interpreter and compiler.

Reply:
One case I remember is wxWidgets.
Might be the only relevant case right now, but that could change with new packages.
Anyway, it’s just a hint printed out saying that there is something which should be checked.

Reply:
Too much false positive will make people ignore warning.
If no other packages have problem with it, I don't think it's worth to warrant a warning.

I'd like to hear other's opinion. :)

Reply:
Fair point. :)